### PR TITLE
46 npm install error

### PR DESCRIPTION
**Pull request by Dagg-h**
### Issue number

#46 

### Description of fix

Install error fixed. Still a lot of warnings but will make another ticket for it to look at it a different day.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -3,7 +3,12 @@
     "browser": true,
     "es6": true
   },
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:prettier/recommended",
```
Comment (patrickswijgman) [inline .eslintrc.json:8]: Don't forget to add this to the config as well:
```
  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": "error"
  }
```
To enable the prettier plugin :) which means you also need to install this plugin: https://www.npmjs.com/package/eslint-plugin-prettier

Also, as stated in their installation steps; `"plugin:prettier/recommended"` works best if it's the only item in `extends`. But we also need the typescript ones for obvious reasons :stuck_out_tongue: so the two might have conflicts but you can try it out and see if everything still works accordingly :slightly_smiling_face:

File: src/transport.ts [hunk 2]
```diff
@@ -95,23 +95,23 @@ const CANCELLED_REASON = {
 // tslint:disable-next-line: max-classes-per-file
```
Comment (patrickswijgman) [inline src/transport.ts:95]: Do these `tslint` comments still work with eslint?

File: .eslintrc.json [hunk 3]
```diff
@@ -3,7 +3,12 @@
...
+    "plugin:@typescript-eslint/eslint-recommended",
```
Comment (patrickswijgman) [inline .eslintrc.json:9]: Also add `'@typescript-eslint'` to the plugins

**Comment by Vialer-integrations** [comment]
test

File: .eslintrc.json (hunk 1 above)
Comment (Vialer-integrations) [reply .eslintrc.json:8]: Thx, fixed.

File: src/transport.ts (hunk 2 above)
Comment (Dagg-h) [reply src/transport.ts]: nope, removed them.

File: .eslintrc.json (hunk 3 above)
Comment (Dagg-h) [reply .eslintrc.json:9]: check :)

**Comment by patrickswijgman** [comment]
LGTM! :+1:

**Pull request merged**